.eslintrc.yaml

```diff
@@ -0,0 +1,70 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
+  no-extend-native: [2]
+  no-floating-decimal: [2]
+  no-lone-blocks: [2]
+  no-loop-func: [2]
+  no-multi-spaces: [2]
+  no-native-reassign: [2]
+  no-new-wrappers: [2]
+  no-return-assign: [2]
+  no-self-compare: [2]
+  radix: [2]
+  yoda: [2, "never"]
+
+  # Strict Mode
+
+  strict: [2, "never"]
+
+  # Variables
+
+  no-shadow: [2]
+  no-use-before-define: [2]
+
+  # Stylistic Issues
+
+  block-spacing: [2, "always"]
+  brace-style: [2, "1tbs", allowSingleLine: true]
+  camelcase: [2, properties: "always"]
+  comma-style: [2, "last"]
+  consistent-this: [2, "self"]
+  eol-last: [2]
+  indent: [2, 2]
```

Comment:
I'd recommend we change this to indent: [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}] (see http://eslint.org/docs/rules/indent.html) because that allows this:
```
let x = 0,
    y = 2,
    z = loader.value;

const chickens = "TASTY",
      breakfast = require('cereals').CORNFLAKES;

var production = false,
    mocks3 = true,
    config = require('./lib/config');
```